### PR TITLE
Newsletter-270-Translate-into-French 

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2023-09-27-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+   reste bloqué, cela peut être très ennuyeux. Il est également possible de sonder plusieurs chemins simultanément et d'utiliser les
+   résultats pour choisir le meilleur chemin quelques instants plus tard lors de l'envoi d'un paiement. {% assign timestamp="53:01" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="28492,119,738,28246,6311,6617,6686,2613,7994,2547,2534" %}
+[LND v0.17.0-beta.rc5]: https://github.com/lightningnetwork/lnd/releases/tag/v0.17.0-beta.rc5
+[news253 ark]: /fr/newsletters/2023/05/31/#proposition-d-un-protocole-de-gestion-des-joinpool
+[maxwell clock stop]: https://www.reddit.com/r/Bitcoin/comments/37fxqd/it_looks_like_blockstream_is_working_on_the/crmr5p2/
+[law cov post]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-September/004092.html
+[law cov paper]: https://github.com/JohnLaw2/ln-scaling-covenants[towns cov]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-September/004095.html
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-09-27-newsletter.md:201]:
```suggestion
[law cov paper]: https://github.com/JohnLaw2/ln-scaling-covenants
[towns cov]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-September/004095.html
```

**Review by murchandamus — COMMENTED** [review]
Looks good, I just noticed one broken link:

File: _posts/fr/newsletters/2023-09-27-newsletter.md (hunk 1 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2023-09-27-newsletter.md:201]: Oh ! thanks for this correction @murchandamus

File: _posts/fr/newsletters/2023-09-27-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    financeur de l'usine de déplacer des fonds pour les utilisateurs occasionnels d'une usine (basée sur une transaction onchain
+    particulière) vers une autre usine (ancrée dans une transaction onchain différente) sans nécessiter d'interaction de l'utilisateur.
+    Cela signifie que l'utilisateur occasionnel Alice, qui sait qu'elle doit se connecter avant l'expiration de 6 mois d'une usine,
+    peut se connecter au mois 5 pour découvrir que ses fonds ont déjà été transférés vers une nouvelle usine avec plusieurs mois
+    supplémentaires avant l'expiration. Alice n'a rien à faire ; elle conserve un contrôle complet et sans confiance de ses fonds.
+    Cela réduit la possibilité qu'Alice se connecte très près de l'expiration, découvre que le financeur de l'usine est temporairement
+    indisponible et soit obligée de mettre sa part de l'arbre de temporisation onchain, ce qui entraîne des frais de transaction et
+    réduit la scalabilité globale du réseau.
+
+    Anthony Towns a répondu avec une préoccupation qu'il a appelée le problème du "troupeau tonitruant" (appelé "spam d'expiration
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-09-27-newsletter.md:56]:
```suggestion
    Anthony Towns a [répondu][towns cov] avec une préoccupation qu'il a appelée le problème du "troupeau tonitruant" (appelé "spam d'expiration
```

File: _posts/fr/newsletters/2023-09-27-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    forcée" dans le document LN original) où l'échec délibéré ou accidentel d'un grand utilisateur dédié nécessite à de nombreux
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-09-27-newsletter.md:57]:
```suggestion
    forcée" dans le document [LN original][ln paper]) où l'échec délibéré ou accidentel d'un grand utilisateur dédié nécessite à de nombreux
```

File: _posts/fr/newsletters/2023-09-27-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    autres utilisateurs de mettre de nombreuses transactions sensibles au temps onchain en même temps. Par exemple, une usine avec un
+    million d'utilisateurs peut nécessiter une confirmation sensible au temps jusqu'à un million de transactions plus une confirmation
+    non sensible au temps jusqu'à deux millions de transactions supplémentaires pour que ces utilisateurs replacent ces fonds dans de
+    nouveaux canaux. Il faut actuellement environ une semaine au réseau pour confirmer trois millions de transactions, donc les
+    utilisateurs d'une usine d'un million d'utilisateurs peuvent souhaiter qu'une usine renouvelle leurs fonds quelques semaines avant
+    l'expiration, ou peut-être plusieurs mois à l'avance s'ils sont préoccupés par le fait que plusieurs usines de plusieurs millions
+    d'utilisateurs rencontrent des problèmes simultanément.
+
+    Une version du document LN original suggérait que ce problème pourrait être résolu en utilisant une idée de Gregory Maxwell qui
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-09-27-newsletter.md:66]:
```suggestion
    Une version du document LN original suggérait que ce problème pourrait être résolu en utilisant une [idée][maxwell clock stop] de Gregory Maxwell qui
```

File: _posts/fr/newsletters/2023-09-27-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+    retarderait l'expiration lorsque "les blocs sont pleins" (par exemple, les frais sont supérieurs au montant normal). Dans sa
+    réponse à Towns, Law a noté qu'il travaille sur une conception spécifique pour une solution de ce type qu'il publiera une fois
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-09-27-newsletter.md:68]:
```suggestion
    [réponse][law fee stop] à Towns, Law a noté qu'il travaille sur une conception spécifique pour une solution de ce type qu'il publiera une fois
```

File: _posts/fr/newsletters/2023-09-27-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+
+- [Existe-t-il un moyen de télécharger les blocs à partir de zéro sans télécharger d'abord les en-têtes de bloc ?]({{bse}}119503)
+  Pieter Wuille confirme qu'il est possible de télécharger les blocs sans les en-têtes, mais souligne que le désavantage est que le
+  nœud ne saurait pas s'il est sur la meilleure chaîne avant d'avoir téléchargé et traité tous les blocs. Il compare cette approche
+  avec [la synchronisation des en-têtes en premier][headers first pr] et décrit les messages P2P échangés pour chaque approche.
+  {% assign timestamp="26:56" %}
+
+- [Où est indiqué le plafond dur de 21 millions de bitcoins dans le code source de Bitcoin ?]({{bse}}119475)
+  Pieter Wuille explique la fonction `GetBlockSubsidy` de Bitcoin Core qui définit le calendrier d'émission des subventions. Il renvoie
+  également à une discussion précédente sur Stack Exchange concernant la limite de 20 999 999,9769 BTC de Bitcoin et pointe vers la
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-09-27-newsletter.md:101]:
```suggestion
  également à une discussion précédente sur Stack Exchange concernant la [limite de 20 999 999,9769 BTC]({{bse}}38998) de Bitcoin et pointe vers la
```

File: _posts/fr/newsletters/2023-09-27-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+
+- [LND #7994][] ajoute la prise en charge de l'interface RPC du signataire distant pour l'ouverture de canaux taproot, ce qui nécessite
+  de spécifier une clé publique et le nonce en deux parties [MuSig2][topic musig]. {% assign timestamp="50:18" %}
+
+- [LDK #2547][] met à jour le code de recherche de chemin probabiliste en supposant qu'il est plus probable que les canaux distants
+  aient la plupart de leur liquidité poussée d'un côté du canal. Par exemple, dans un canal distant de 1,00 BTC entre Alice et Bob,
+  l'état le moins probable du canal est que Alice et Bob aient chacun 0,50 BTC. Il est plus probable que l'un d'entre eux ait 0,90
+  BTC---et encore plus probable que l'un d'entre eux ait 0,99 BTC. {% assign timestamp="51:10" %}
+
+- [LDK #2534][] ajoute la méthode `ChannelManager::send_preflight_probes` pour sonder les chemins de paiement avant d'essayer d'envoyer
```
Comment (murchandamus) [inline _posts/fr/newsletters/2023-09-27-newsletter.md:180]:
```suggestion
- [LDK #2534][] ajoute la méthode `ChannelManager::send_preflight_probes` pour [sonder][topic payment probes] les chemins de paiement avant d'essayer d'envoyer
```

**Review by bitschmidty — APPROVED** [review]
ACK, thanks for the translation @Copinmalin !

**Pull request merged**